chartmuseum.yaml

```diff
@@ -1,40 +1,149 @@
 package:
   name: chartmuseum
   version: 0.16.2
-  epoch: 1
+  epoch: 2
   description: helm chart repository server
   copyright:
     - license: Apache-2.0
-  dependencies:
-    runtime:
-      - ca-certificates-bundle
 
 environment:
-  contents:
-    packages:
-      - bash
-      - build-base
-      - busybox
-      - ca-certificates-bundle
-      - go
+  environment:
+    CGO_ENABLED: 0
```

Comment:
A comment here would be useful as to why this is set - i.e. "lots of deps have cgo accelerations, which chartmuseum builds typically opt out of".